backend/src/app.js

```diff
@@ -161,6 +167,36 @@ app.get('/api/v1/legislators/search/offices', async (req, res, next) => {
   }
 });
 
+app.get(
+  '/api/v1/geocoding/:loc',
+  async (req, res, next) => {
+    const url = mapBoxApi(
+      `geocoding/v5/mapbox.places/${req.params.loc}.json`,
+      {
+        country: 'US',
+        fuzzyMatch: true
+      }
+    );
+
+    const apiResponse = await fetch(url);
+    console.log(apiResponse);
+    const out = await apiResponse.json();
+    console.log(out);
+    if (!out) {
+      // TODO: upgrade expressJS when v5 is stable
+      // https://expressjs.com/en/guide/error-handling.html
+      console.log('Failed geocode request:');
+      console.log(out.message);
+      console.log(url);
+      next(
+        'Did not successfully retrieve lat/long from Mapbox'
+      );
+    } else {
+      res.json(out);
+    }
+  }
+);
+
```

Comment:
Would love a review of this code @agreffard @elliott-king . In the new PR, I'll clean this up from console.log(url) and out.message as they aren't needed anymore, or we can simply do that now :)

Reply:
Let's clean it now, but it lgtm otherwise